site.yml

```diff
@@ -119,6 +119,12 @@ apps:
     spec: osymandius/csavr_mortality
     admin:
       - ostevens
+
+  naomi_viewer:
```

Comment:
I know we're not terribly consistent, but `naomi-viewer` would be a more typical url; could you change this line please? The repo itself is fine.

Reply:
Thanks - resolved. Hope to have UNAIDS bring it into the fold at xxxx.unaids.org as per naomi & shiny90.

Reply:
if that's the case should we move this repo into mrc-ide?